packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockEmojiString.st

```diff
@@ -0,0 +1,10 @@
+emojis
+mockEmojiString
+	
+	"A string containing 'A:rainbow flag::face with tears of joy:C:transgender flag'"
+	"We would love to display them here, but Squeak does not support emojis"
+	^ self mockEmojiStringValues
+		inject: ''
```

Comment:
You used `String new` instead of `''` in `TCUEmojiHelper>>fileNameFor:`. Dou you think it should be kept consistent?